ios/src/TGMapViewController.mm

```diff
@@ -519,6 +519,7 @@ - (void)respondToShoveGesture:(UIPanGestureRecognizer *)shoveRecognizer
 - (void)viewDidLoad
 {
     [super viewDidLoad];
+    NSLog(@"Font families: %@", [UIFont familyNames]);
```

Comment:
If we want to log from the SDK side, we should probably add a macro to not output the log statements from release builds of the framework.

Otherwise you should remove this.